x-pack/plugins/cases/public/components/app/use_available_owners.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useKibana } from '../../common/lib/kibana';
+
+/**
+ *
+ * @param level : 'crud' | 'read' (default: 'crud')
+ *
+ * `securitySolution` owner uses cases capability feature id: 'securitySolutionCases'; //owner
+ * `observability` owner uses cases capability feature id: 'observabilityCases';
+ * both solutions use `crud_cases` and `read_cases` capability names
+ **/
+
+export const useAvailableCasesOwners = (level: 'crud' | 'read' = 'crud'): string[] => {
+  const { capabilities } = useKibana().services.application;
+  const capabilityName = `${level}_cases`;
+  return Object.entries(capabilities).reduce(
+    (availableOwners: string[], [featureId, capability]) => {
+      if (featureId.endsWith('Cases') && !!capability[capabilityName]) {
+        availableOwners.push(featureId.replace('Cases', ''));
+      }
+      return availableOwners;
+    },
+    []
+  );
```

Comment:
not necessary, but It would be nice to ensure this array has always the same order.
```suggestion
  ).sort();
```

Reply:
i thought about that too, but it seemed like either way, if we add something to the array, it adds the possibility that the index of something changed, if we were referencing it by location we'd have to change it elsewhere. I think to avoid this, the function caller should always have to search through the returned array.

Reply:
Yes, you are right. It was only to make sure it is always the same array, just in case it is iterated somewhere else to render something in the future. nevermind, it is not needed in the current implementation 👍